bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-resources",
-  "version": "2.0.7",
+  "version": "2.0.8",
```

Comment:
As per https://github.com/adaptlearning/adapt-contrib-slider/pull/124#pullrequestreview-50083087, we're now leaving the versioning up to the person who does the release, so I wouldn't worry about bumping in this PR.

In this case it'll probably be changing to `2.1.0` anyway since we're introducing a new feature.

Note to whoever releases: we'll also need to bump the framework dependency up to the (future) version that includes the amends to Drawer (https://github.com/adaptlearning/adapt_framework/pull/1685).